tests/test_mlp.py

```diff
@@ -1,7 +1,8 @@
 import unittest
 
 import torch
-from vissl.models.heads import LinearEvalMLP, MLP
+
```

Comment:
I am curious about the additional lines. Do you know isort is enforcing this?

Reply:
After your comment above, I figured it may have been influenced by my removing the `known_third_party` field. So I reverted the commit `22cf0d4` that applied the changes to other files and reintroduced the upgrade with the exception of removing `known_third_party`. These new lines were still applied to the original codebase, so it seems it is indeed something `isort` is enforcing. 

Since the history with the reverted commit would just reintroduce the changes, I have instead pushed a simple new commit to restore the original `setup.cfg`, since it appears to have no effect on the sorting. I now remember that I reasoned that since `known_third_party` was generated, and that since we were no longer generating anything to update it, that I would remove it from the codebase to avoid a stagnant line.